inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx

```diff
@@ -49,6 +54,7 @@ export class ToolsTabComponent extends PaneComponent {
     private _reflectorHostname: string = "localhost";
     private _reflectorPort: number = 1234;
     private _reflector: Reflector;
+    private _envOptions = { imageType: 0, imageQuality: 0.8 };

```

Comment:
I guess imageType should be image/png by default :-)

Reply:
Also is imageQuality 0.8 by default on png ? I thought the quality was depending on the mime type for the default value ?

Reply:
> I guess imageType should be image/png by default :-)

It is. I used a numeric value for `imageType`, as it seems `<OptionsLineComponent>` can only deal with numbers as value, right? So this numbers refers to the index of ´envExportImageTypes´, and ´envExportImageTypes[0]´ is the entry for PNG!

> Also is imageQuality 0.8 by default on png

I believe PNG will ignore the quality all together, as it only supports lossless compression.

Reply:
I see :-) it is just naming then, could you rename imageType in _envOptions to imageTypeIndex or something similar to prevent any confusion ? 

Basically looking into it too quickly I tought it was used as the string here.